unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerHandler.cs

```diff
@@ -464,5 +437,31 @@ void RemoveActiveColliders(IList<Collider> colliders, IInternalECSComponent<Inte
 
             dataStoreEcs7.RemovePendingResource(scene.sceneData.sceneNumber, gltfSrc);
         }
+
+        private void AddCurrentRendereableToSceneMetrics()
+        {
+            DataStore.i.sceneWorldObjects.AddRendereable(entity.scene.sceneData.sceneNumber, currentRendereable);
+
+            foreach (Renderer renderer in renderers)
+            {
+                foreach (Material material in renderer.sharedMaterials)
+                {
+                    DataStore.i.sceneWorldObjects.AddMaterial(entity.scene.sceneData.sceneNumber, entity.entityId, material);
```

Comment:
This class has already some data stores being injected, can we do the same with this one?

Reply:
Sounds good, done!